### PR TITLE
add missing hibernate envers dependency

### LINKED ISSUE #3010: Mongo DB & Cassandra build fails [master]
Our Mongo DB builds in travis are failing with below

```
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/MongoBankAccountDTO.java:9: error: cannot find symbol
import javax.persistence.Lob;
                        ^
  symbol:   class Lob
  location: package javax.persistence
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:9: error: cannot find symbol
import javax.persistence.Lob;
                        ^
  symbol:   class Lob
  location: package javax.persistence
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/MongoBankAccountDTO.java:52: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.MongoBankAccountDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/MongoBankAccountDTO.java:57: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.MongoBankAccountDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:137: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:143: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:149: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:155: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:160: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:166: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:172: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:178: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:183: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:188: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:193: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
[ERROR] diagnostic: /home/travis/app-mongodb/src/main/java/com/mycompany/myapp/web/rest/dto/FieldTestMapstructEntityDTO.java:198: error: cannot find symbol
    @Lob
     ^
  symbol:   class Lob
  location: class com.mycompany.myapp.web.rest.dto.FieldTestMapstructEntityDTO
```

### PR DESCRIPTION
**Pull request by ruddell**
This dependency disappeared recently from _pom.xml

Fix #3010

### CODE REVIEW

**Comment by jdubois** [comment]
OK, but why on earth do we need Envers? We are not using it.

**Comment by jdubois** [comment]
Sorry, I know why, it's for the security audits...

**Comment by jdubois** [comment]
So this all OK for me!!

File: generators/server/templates/_pom.xml [hunk 1]
```diff
@@ -377,6 +377,10 @@
             <groupId>org.hibernate</groupId>
             <artifactId>hibernate-validator</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-envers</artifactId>
+        </dependency>
         <%_ if (databaseType == 'sql') { _%>
         <dependency>
```
Comment (deepu105) [inline generators/server/templates/_pom.xml]: this should be removed

**Comment by deepu105** [comment]
same for gradle as well

**Comment by deepu105** [comment]
those failures are annoying so im merging this and fixing gradle as well

**Pull request merged**